demo/ksnav_app_linux/Dockerfile_ksnav

```diff
@@ -0,0 +1,123 @@
+#BUILDER 
```

Comment:
Do we need this custom file or can we use what is already in https://github.com/elisa-tech/ks-nav/tree/main/container  ?

Reply:
Merge with original

Reply:
Merged demo updates with original.
Added an argument to enable Demo mode (original instructions use a remote database - while the demo uses a local db). It is enabled by appending --build-arg _EN_DEMO=true to the normal build command. 
Omitting to add this argument to the build command will return the 'nominal' original scenario.